frontend/scripts/inject-analytics.js

```diff
@@ -0,0 +1,13 @@
+// Inject analytics scripts into the page
```

Comment:
Do we even want this example file anymore? Someone could use a different analytics system than plausible, but I suppose if we want it as an example, could just add it into the docs?

Reply:
Sure, yeah, and I figure it's probably useful for us to have too? Idk, we could get rid of it too, the plausible injection script is pretty simple, we can redo it if needed

Reply:
I'll add to the docs.